xm-commons-idp/HELP.md

```diff
@@ -0,0 +1,3 @@
+## xm-commons-idp
+XM^online commons project for XM authentication context. 
+// FIXME: rename file to README.md if it will have some useful content or delete
```

Comment:
@olye0216 please resolve fixme - rename file to README.md

Reply:
Remark applied.